server/routes/overview/index.ts

```diff
@@ -27,7 +28,7 @@ export default (router: Router, services: Services) => {
   const workAndInterestsController = new WorkAndInterestsController()
   const educationAndTrainingController = new EducationAndTrainingController(services.curiousService)
   const viewArchivedGoalsController = new ViewArchivedGoalsController(services.educationAndWorkPlanService)
-  const viewGoalsController = new ViewGoalsController()
+  const viewGoalsController = new ViewGoalsController(services.educationAndWorkPlanService)
```

Comment:
You dont need to do this; no need to inject the service into the controller (see my other comments)